app/web_framework/cluster_base_handler.py

```diff
@@ -62,6 +69,24 @@ def _handle_request_exception(self, ex):
         self.set_status(status_code)
         self.finish()
 
+    # For methods that annotate with `Tuple`, use `disable=invalid-sequence-index`
+    # This bug is fixed in https://github.com/PyCQA/astroid/commit/563031aaf13a44adc8db4f8d0ab8020d550aae00
+    # More information on the issue in https://github.com/PyCQA/pylint/issues/1212
+
+    def get_pagination_params(self) -> Tuple[int, int]:  # pylint: disable=invalid-sequence-index
+        offset = self.get_query_argument('offset', pagination_constants['DEFAULT_OFFSET'], True)
+        limit = self.get_query_argument('limit', pagination_constants['DEFAULT_LIMIT'], True)
+        return self._validate_arguments(int(offset), int(limit))
+
+    @staticmethod
+    def _validate_arguments(offset: int, limit: int) -> Tuple[int, int]:  # pylint: disable=invalid-sequence-index
+        offset = offset or pagination_constants['DEFAULT_OFFSET']
+        limit = limit or pagination_constants['DEFAULT_LIMIT']
+        # Negative offsets or limits are not allowed
+        non_negative_offset = max(offset, 0)
```

Comment:
@josephharrington Hah yeah good point, that would get really confusing. I was thinking more like `offset=-1&limit=20` for 10 builds would give builds 9 to 10, then 1 to 9 etc so the wrap wouldn't repeat builds, but yeah like you said it'd get really confusing and I don't think this would be a good idea after I think about it